tests/test_test_utils.py

```diff
@@ -134,6 +133,14 @@ async def test_get_route() -> None:
         await test_get_route()
 
 
+def test_unittest_run_loop() -> None:
+    with pytest.warns(DeprecationWarning):
```

Comment:
In `pytest.warns` and `pytest.raises`, we should always use `match=r'...'`

```suggestion
    with pytest.warns(DeprecationWarning, match=r"Decorator no longer needed in 3\.8\+"):
```